frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/transaction_handler/editor_transaction_handler.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:appflowy/plugins/document/presentation/editor_plugins/plugins.dart';
+import 'package:flutter/material.dart';
+
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+const List<String> nodeTypesContainingMentions = [
```

Comment:
Is it possible to use a function to generate the types? As far as I know, the node that contains delta should be able to contain the mention